src/component/NavbarCustom.js

```diff
@@ -1,35 +1,64 @@
 import React from "react";
-import { Container, Navbar, Nav, NavDropdown } from "react-bootstrap";
+import { Container, Navbar, Nav } from "react-bootstrap";
+
+const navigationItems = {
+    Cabinet: ["Presentation", "Equipe", "Missions"],
+    Formation: ["Catalogue", "Sur mesure"]
+};
 
 class NavbarCustom extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            hasSubMenu: false,
+            activeSubMenu: "",
+        };
+        this.setHasSubMenu = this.setHasSubMenu.bind(this);
 
+    }
+    setHasSubMenu(newActiveSubMenu) {
+        this.setState({ activeSubMenu: this.state.hasSubMenu && (this.state.activeSubMenu === newActiveSubMenu)? "" : newActiveSubMenu, hasSubMenu: this.state.hasSubMenu && (this.state.activeSubMenu !== newActiveSubMenu)? true : !this.state.hasSubMenu });
```

Comment:
La ligne est un peu longue, tu pourrais faire
this.setState({ 
activeSubMenu: this.state.hasSubMenu && (this.state.activeSubMenu === newActiveSubMenu)? "" : newActiveSubMenu, 
  hasSubMenu: this.state.hasSubMenu && (this.state.activeSubMenu !== newActiveSubMenu)? true : !this.state.hasSubMenu 
});

Reply:
yass grl